subprojects/ide/ide.gradle

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+apply from: "$rootDir/gradle/tapiTest.gradle"
```

Comment:
⭕️ Should be automatically applied in `groovyProject.gradle` if a project has a `src/tapiTest` folder